src/main/java/mate/academy/bookshop/controller/BookController.java

```diff
@@ -54,7 +54,7 @@ public List<BookDto> searchBooks(BookSearchParametersDto searchParameters, Pagea
 
     @Operation(summary = "Create a new book",
             description = "Create a new book")
-    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @PreAuthorize("hasRole('ADMIN')")
```

Comment:
why on user accessible endpoints you did not make this update?

Reply:
discussed on Q&A